packages/ckeditor5-paste-from-office/src/normalizers/mswordnormalizer.ts

```diff
@@ -22,13 +23,16 @@ const msWordMatch2 = /xmlns:o="urn:schemas-microsoft-com/i;
 export default class MSWordNormalizer implements Normalizer {
 	public readonly document: ViewDocument;
 
+	public readonly editor: Editor;
```

Comment:
Do we need `editor` here? Why don't just store only information that we need, i.e. `hasMultiLevelListPlugin`?